frontend/src/metabase/query_builder/components/NativeQueryEditor/VisibilityToggler/VisibilityToggler.jsx

```diff
@@ -29,7 +29,7 @@ const VisibilityToggler = ({
   );
 
   return (
-    <Container>
+    <Container className="hide sm-show">
```

Comment:
Can you make sure this is really needed? As `classNames` in the line below will already contain `hide sm-show` when the button is supposed to be visible, this causes the parent and its immediate child to have the same `hide sm-show` class as seen in the screenshot below. This also [broke the unit test](https://github.com/metabase/metabase/runs/7739578616?check_suite_focus=true).

https://github.com/metabase/metabase/blob/7350ce03b3a2150bf67d9697acf8a5724886bb37/frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx#L447-L448

![image](https://user-images.githubusercontent.com/1937582/183597475-5d40969c-033c-4c03-a52b-381e00acf1e2.png)

Reply:
Yeah, because it was in the line below, we still show that container with 8px of padding. I think I will try and use emotion to just remove the padding on mobile.

Reply:
Actually, this was from one of the earlier design ideas, and isn't even needed anymore. Removing the classnames all together